Modules/System Test Libraries/XML Validation/app.json

```diff
@@ -0,0 +1,25 @@
+{
+    "id": "5a44012f-8332-4234-9b67-1e1e5090135f",
+    "name": "XML Validation Test Library",
```

Comment:
Test libraries are usually used to avoid dependencies between test modules. 

In this case, it is needed. Helper functions in the test module itself would do the trick.

Reply:
Removed test library and added helper codeunit to the test module.